tests/route-spec/responses.test.ts

```diff
@@ -0,0 +1,273 @@
+import test from "ava"
+import { createWithEdgeSpec } from "src/create-with-edge-spec"
+import { EdgeSpecRequest, EdgeSpecResponse } from "src/types/web-handler"
+import { getTestRoute } from "../fixtures/get-test-route.js"
+import { z } from "zod"
+import { GlobalSpec } from "src/types/global-spec.js"
+
+const defaultSpecs = {
+  globalSpec: {
+    apiName: "hello-world",
+    productionServerUrl: "https://example.com",
+
+    exceptionHandlingMiddleware: async (next, req) => {
+      try {
+        return await next(req)
+      } catch (e: any) {
+        return EdgeSpecResponse.json(
+          { error_type: e.constructor.name },
+          { status: 500 }
+        )
+      }
+    },
```

Comment:
do we still need an explicit option for this now that middlewares can internally `await next()`?

Reply:
Idk, why did we have it in nextlove?

Reply:
I think it's old cruft that was never removed. Related to the default behavior of adding `ok: true` to every response I think.